package/network/services/lldpd/files/lldpd.init

```diff
@@ -339,6 +322,6 @@ reload_service() {
 }
 
 stop_service() {
-	rm -rf ${LLDPD_RUN} 2>/dev/null
+	rm -rf ${LLDPD_RUN} $LLDPSOCKET
```

Comment:
Perhaps the `2>/dev/null` redirect should be retained here?

Reply:
Wouldn't this mean that if an error occurs when the service is stopped, the user would not know about it?